tests/ssl/generate_tls_keys.sh

```diff
@@ -109,13 +109,11 @@ gen_crl() {
         -out "${OUT_DIR}/$signer/crl.pem"
 }
 
-declare -a names=(mysql postgresql acra-writer acra-writer-2 
-    acra-writer-revoked acra-server ocsp-responder intermediate-ca vault 
-    acra-client redis)
+declare -a names=(consul)
 for name in "${names[@]}"; do
     gen_cert ca $name
 
-    if [[ "$name" = "vault" ]]; then
+    if [ "$name" = "vault" ] || [ "$name" = "consul" ] ; then
```

Comment:
for what we generate PEM certs too? because of python's clients require it? if so then can we leave comment here for what we do that only for them?

Reply:
Python client require a simple crt format. Its required for Vault/Consul servers to configure, anyway will let the comment.